LayoutTests/svg/repaint/image-srcset-transform-animation-zoom.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<style>
+    body { zoom: 2; margin: 0; padding: 0; }
+</style>
+<body>
+    <picture>
+        <source srcset="../../resources/move.svg 2x"></source>
+        <img id="svgImg" alt="" />
+    </picture>
+</body>
+<script>
+    if (window.testRunner) {
+        testRunner.waitUntilDone();
+        if (testRunner.dontForceRepaint)
+            testRunner.dontForceRepaint();
+    }
+
+    async function main() {
+        setTimeout(() => {
+            if (window.testRunner)
+                testRunner.notifyDone();
+        }, 250);
```

Comment:
Maybe a comment saying that this matches the duration of the animation in the SVG.

Reply:
The SVG uses 100ms.